src/main/java/org/spongepowered/api/block/tile/Tile.java

```diff
@@ -48,4 +48,24 @@
      */
     BlockLoc getBlock();
 
+    /**
+     * Checks for whether the tile is currently valid or not.
```

Comment:
No, it really isn't. It's literally just a boolean stored in the class itself that is sometimes called by classes outside. The only example I could find was in BlockFurnace.